pyhealth/datasets/base_dataset.py

```diff
@@ -226,14 +226,22 @@ def load_table(self, table_name: str) -> pl.LazyFrame:
         if timestamp_col:
             if isinstance(timestamp_col, list):
                 # Concatenate all timestamp parts in order with no separator
+                logger.info(
+                    f"Combining multiple timestamp columns: {timestamp_col}"
+                    f" with format: {timestamp_format}"
+                )
                 combined_timestamp = pl.concat_str(
                     [pl.col(col) for col in timestamp_col]
-                ).str.strptime(pl.Datetime, format=timestamp_format, strict=True)
```

Comment:
Any reason why we should set `strict=False`? I think we might want to be a little bit cautious with this as our internal data structure is based on event sequence ordered by timestamp.